app/stores/AppStore.js

```diff
@@ -1,17 +1,51 @@
 // @flow
 import { observable, action, computed } from 'mobx';
+import { defineMessages } from 'react-intl';
 import Store from './lib/Store';
 import CachedRequest from './lib/CachedRequest';
 
+const messages = defineMessages({
+  languageOptionEnglish: {
+    id: 'profile.languageSelect.form.options.english',
+    defaultMessage: '!!!English',
+    description: 'Label for the language select option "English".',
+  },
+  languageOptionJapanese: {
+    id: 'profile.languageSelect.form.options.japanese',
+    defaultMessage: '!!!Japanese',
+    description: 'Label for the language select option "Japanese".',
+  },
+  languageOptionChinese: {
+    id: 'profile.languageSelect.form.options.chinese',
+    defaultMessage: '!!!Chinese',
+    description: 'Label for the language select option "Chinese".',
+  },
+  languageOptionKorean: {
+    id: 'profile.languageSelect.form.options.korean',
+    defaultMessage: '!!!Korean',
+    description: 'Label for the language select option "Korean".',
+  },
+  languageOptionGerman: {
+    id: 'profile.languageSelect.form.options.german',
+    defaultMessage: '!!!German',
+    description: 'Label for the language select option "German".',
+  },
+  languageOptionCroatian: {
+    id: 'profile.languageSelect.form.options.croatian',
+    defaultMessage: '!!!Croatian',
+    description: 'Label for the language select option "Croatian".',
+  },
+});
```

Comment:
Thanks for the tip! Global messages are much much better place to put these translations ;)